crates/symbolicator/src/config.rs

```diff
@@ -420,6 +420,44 @@ mod tests {
         )
     }
 
+    #[test]
+    fn test_unspecified_dl_timeouts() {
+        let yaml = r#"
+            sources: []
+        "#;
+        let cfg = Config::from_reader(yaml.as_bytes()).unwrap();
+        let default_cfg = Config::default();
+        assert_eq!(cfg.max_download_timeout, default_cfg.max_download_timeout);
+        assert_eq!(cfg.connect_timeout, default_cfg.connect_timeout);
+        assert_eq!(cfg.streaming_timeout, default_cfg.streaming_timeout);
+    }
+
+    #[test]
+    fn test_zero_second_dl_timeouts() {
```

Comment:
I'd like to double check to make sure this is actually intentional or desired. If I understand things (ie tokio's timeout implementation) correctly, these timeout ASAP. Do we want to allow this? 

The alternative would be to set these to their default values if they're initialized to 0s, but it does open up the question of how low timeouts are allowed to be in a config before we start replacing them with the default value. Allowing 0 here does let us dodge this problem.